codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHTTPBindingProtocolGenerator.kt

```diff
@@ -36,8 +36,8 @@ abstract class AWSHTTPBindingProtocolGenerator(
     val requestTestBuilder = HttpProtocolUnitTestRequestGenerator.Builder()
     val responseTestBuilder = HttpProtocolUnitTestResponseGenerator.Builder()
     val errorTestBuilder = HttpProtocolUnitTestErrorGenerator.Builder()
-    open val testsToIgnore: Set<String> = setOf()
-    open val tagsToIgnore: Set<String> = setOf()
+    open val protocolTestsToIgnore: Set<String> = setOf()
+    open val protocolTestTagsToIgnore: Set<String> = setOf()
```

Comment:
Small rename for specifying type of tests these apply to, just for additional clarity.